### ISSUE TITLE
not found latest release version docker image in docker hub

### ISSUE THREAD

**Issue by Wenfeng-GAO** [argoproj/argo-workflows#3004]
# Summary
Now the latest release version is v2.8.0, and when follow the **getting-started** tutorial
```bash
kubectl create namespace argo
kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/install.yaml
```
I found that the controllers use images `argoproj/workflow-controller:v2.8.0` and `argoproj/argocli:v2.8.0`, however version v2.8.0 is missed in docker hub.

Do we have some automation or plans to make docker hub image keep up with the binary releaes version?

**Comment by alexec** [comment]
The build scripts seemed to have used an old tag. I'll re-push.

**Comment by alexec** [comment]
```
$ make status# GIT_TAG=stable, GIT_BRANCH=release-2.8, GIT_TREE_STATE=clean, MANIFESTS_VERSION=v2.8.0-rc4, VERSION=v2.8.0-rc4, DEV_IMAGE=false
[acollins8@intuitdepe1a95 argo (⎈ |k3s-default:argo) (release-2.8)]$ git log -n2commit 8f696174746ed01b9bf1941ad03da62d312df641 (HEAD -> release-2.8, tag: v2.8.0, tag: stable, origin/release-2.8)
Author: Alex Collins <alex_collins@intuit.com>
Date:   Mon May 11 15:54:30 2020 -0700

    Update manifests to v2.8.0

commit ee0dc575dc7d2187e0e97e768c7b58538958608b (tag: v2.8.0-rc4)
Author: Alex Collins <alex_collins@intuit.com>
Date:   Wed May 6 16:17:02 2020 -0700

    Update manifests to v2.8.0-rc4
```

**Comment by alexec** [comment]
Fixed by be32b207ddf4490f90d7e85df9de84b52b46eb6e (needs PR).

New images pushed.
